apps/src/executionLog.js

```diff
@@ -24,9 +24,16 @@ module.exports.getResultsFromLog = function (logConditions, executionLog) {
      * @param {!Object} condition
      * @param {string} [condition.matchType] 'exact', 'inexact'
      * @param {number} [condition.minTimes] number of matches required
+     * @param {number} [condition.maxTimes] maximum number of matches allowed
      * @param {string[]} [condition.entries] function or statement names
      * @param {string} [condition.message] message to display if condition fails
      */
+
+    condition.minTimes = condition.minTimes || 0;
+    if (typeof condition.maxTimes === 'undefined') {
+      condition.maxTimes = Infinity;
+    }
```

Comment:
Hmmm...that seems bad if we're discouraged from using a utils package because it's harder to test. I'll take a look if I can make this easier.